package.json

```diff
@@ -12,12 +12,11 @@
     "karma": "karma",
     "karma.start": "karma start",
     "lint": "gulp tslint",
-    "postinstall": "tsd reinstall --clean && tsd link && tsd rebundle && gulp postinstall",
     "start": "gulp serve --env dev",
     "serve.dev": "gulp serve --env dev",
     "tasks.list": "gulp --tasks-simple",
     "test": "gulp test",
-    "tsd": "tsd"
+    "tsd": "gulp tsd"
```

Comment:
I disagree with this as `npm run tsd -- <args>` is just a proxy to `tsd` bin for those who don't want to have it installed globally.